bitbots_bringup/scripts/motor_goals_viz_helper.py

```diff
@@ -1,21 +1,41 @@
 #!/usr/bin/env python3
 
+import argparse
+
 import rospy
 from sensor_msgs.msg import JointState
-from trajectory_msgs.msg import JointTrajectory
+from bitbots_msgs.msg import JointCommand
+from humanoid_league_msgs.msg import Animation
 
 class MotorVizHelper:
     def __init__(self):
-        rospy.init_node("motor_viz_helper", log_level=rospy.DEBUG, anonymous=False)
+        parser = argparse.ArgumentParser()
+        parser.add_argument("--w", help="Directly get walking motor goals", action="store_true")
+        parser.add_argument("--a", help="Directly get animation motor goals", action="store_true")
```

Comment:
`add_argument("--walking", "-w", ...)` and `add_argument("--animation", "-a", ...)` would be the more readable and standard confirming way here (this would also mean replacing args.w with args.walking etc in the further code which would improve readability).